pmdarima/pipeline.py

```diff
@@ -123,7 +122,7 @@ def __len__(self):
     @property
     def named_steps(self):
         # Use Bunch object to improve autocomplete
```

Comment:
Comment is no longer needed, since we're not using `Bunch`. Not a blocker, but could be confusing for future contributors